src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: circle, area: " + getArea()
+                + " sq.units, radius: " + getRadius()
+                + " units, color: " + getColor();
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/195878684-81eb400c-fb8d-422a-b9f5-f32b03584260.png)
**`You shouldnt override toString()`**
you should implement this method
fix in each figure